publish/src/commands/deploy/deploy-loans.js

```diff
@@ -14,7 +14,8 @@ module.exports = async ({ account, addressOf, deployer, getDeployParameter, netw
 		args: [account, account, addressOf(ReadProxyAddressResolver)],
 	});
 
-	let WETH_ADDRESS = (await getDeployParameter('WETH_ERC20_ADDRESSES'))[network];
+	// harmony-fix
+	let WETH_ADDRESS = '0xF720b7910C6b2FF5bd167171aDa211E226740bfe' // (await getDeployParameter('WETH_ERC20_ADDRESSES'))[network];
```

Comment:
Probably we should put this and similar addresses in params.json. Hardcoded addresses are hard to find when they are scattered in code